src/coffee/mixins.coffee

```diff
@@ -99,12 +99,8 @@ module.exports =
   ###
   batchList: (list, size) ->
     throw new Error 'List batch size is required' unless size
-    batch = []
-    group = -1
-    for i in [0...list.length]
-      mod = i % size
-      if mod == 0
-        group++
-        batch[group] = []
-      batch[group][mod] = list[i]
-    batch
\ No newline at end of file
+    list.reduce (batch, value, index) ->
+      if index % size == 0 then batch.push []
+      batch[batch.length - 1].push value
+      batch
+    , []
```

Comment:
Is it me or it looks like this line is missing here? `list.reduce (batch, value, index) ->`